test_runner/fixtures/compare_fixtures.py

```diff
@@ -333,3 +336,29 @@ def neon_with_baseline(request: FixtureRequest) -> PgCompare:
     fixture = request.getfixturevalue(request.param)
     assert isinstance(fixture, PgCompare), f"test error: fixture {fixture} is not PgCompare"
     return fixture
+
+
+@pytest.fixture(scope="function", autouse=True)
+def sync_after_each_test():
+    # some of the benchmarks are quite write-happy, and create issues to start
+    # the processes up in 10s.
+    # SYNC_AFTER_EACH_TEST is only "true" for build_and_test workflow
+    # benchmarks, which run test_runner/performance cases sequentially.
+    # however, this fixture is included for all regress tests as well.
+    #
+    # use the environment variable SYNC_AFTER_EACH_TEST to control whether any
+    # syncing happens.
```

Comment:
```suggestion
    # The fixture calls `sync(1)` after each test if `SYNC_AFTER_EACH_TEST` env var is `true`
    # 
    # In CI, `SYNC_AFTER_EACH_TEST` is set to `true` only for benchmarks (`test_runner/performance`) 
    # that are run on self-hosted runners because some of these tests are pretty write-heavy 
    # and create issues to start the processes within 10s
```

Reply:
I'll edit sync(2) :)